### ISSUE TITLE
 When reading keys from "general.el" case is insensitive, eg: "B" will over write "b" label

### ISSUE THREAD

**Issue by dunkarooftop** [justbur/emacs-which-key#204]
When setting up hotkey in "general.el" lower case and upper case will overwrite. for example 

`
 "B"   '(                         :ignore t :which-key "Bookmark")
 "b"   '(                         :ignore t :which-key "BM")
`

Which-key will display both "BM"

**Comment by duianto** [comment]
I'm seeing a similar letter case issue in Spacemacs.

The prefix binding: `SPC b N` is renamed from `+prefix` to `new empty buffer`, here:
https://github.com/syl20bnr/spacemacs/blob/4b195ddfc9228256361e0b264fe974aa86ed51a8/layers/%2Bspacemacs/spacemacs-defaults/keybindings.el#L26

There's also a binding for `SPC b n` which calls `next-buffer`.

But it's which-key name isn't changed, it's just bound here:
https://github.com/syl20bnr/spacemacs/blob/develop/layers/+spacemacs/spacemacs-defaults/keybindings.el#L146

### Issue

The Which-key popup for `SPC b` looks like this:

With the latest version `which-key-20190225.1910`, both bindings are listed as: `new empty buffer`.

![after](https://user-images.githubusercontent.com/13420573/53629555-1a4d6800-3c0e-11e9-9176-f40c9815ba95.png)

With the previous version `which-key-20181114.1432`, only the prefix was renamed, `SPC b n` showed the command name `next-buffer`.

![before](https://user-images.githubusercontent.com/13420573/53629538-115c9680-3c0e-11e9-9dc6-15c75389825d.png)

#### System Info :computer:
- OS: windows-nt
- Emacs: 26.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 4b195ddfc)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(autohotkey emacs-lisp git helm markdown multiple-cursors org spell-checking treemacs version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS LCMS2

**Comment by justbur** [comment]
Thanks, it should be fixed with 51c485f

**Comment by duianto** [comment]
Confirmed, it's working now, thanks.
